compiler/test/fail_compilation/retscope6.d

```diff
@@ -141,6 +141,7 @@ void testarchie()
 
 /* TEST_OUTPUT:
 ---
+fail_compilation/retscope6.d(9015): Error: returning `S9(null).this(r)` escapes a reference to parameter `r`
```

Comment:
This is false. The code below even says:
```D
auto j = fred(&i); // ok
```

Reply:
hmm, you're right.